doc/how_to_upgrade.md

```diff
@@ -10,13 +10,15 @@ We suppose that your configuration is versioned with git, and that it has two re
 
 When typing ```git remote -v``` in your config folder, you should see something like this:
 ```
+cd ~/georchestra
```

Comment:
En fait, ce serait plutôt ```cd ~/myprofile``` comme expliqué là: https://github.com/georchestra/georchestra/blob/14.12/doc/config.md#versioning-the-configuration-directory

Par ailleurs, ton ajout de ligne n'est pas au bon endroit, car il ne s'intègre pas correctement à l'existant.